tests/generic_config_updater/patch_sorter_test.py

```diff
@@ -1787,7 +1788,7 @@ def run_single_success_case(self, data, skip_exact_change_list_match):
         simulated_config = current_config
         for change in actual_changes:
             simulated_config = change.apply(simulated_config)
-            self.assertTrue(ConfigWrapper().validate_config_db_config(simulated_config))
+            self.assertTrue(self.config_wrapper.validate_config_db_config(simulated_config))
```

Comment:
Using a pre-created `config_wrapper` so we don't have to create one each time and was time loading yang models from disk each time.